index.html

```diff
@@ -0,0 +1,104 @@
+<!-- Declare document type as HTML -->
+<!DOCTYPE html>
+<!-- Start HTML document -->
+<html lang="en">
+
+<head>
+    <!-- Declare character set -->
+    <meta charset="UTF-8" />
+    <!-- Declare compatibility with Internet Explorer -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <!-- Declare viewport configuration -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Set title of the webpage -->
+    <title>My Personal Portfolio</title>
+    <!-- Link to external stylesheet -->
+    <link rel="stylesheet" href="css/styles.css" />
+    <!-- Include Fontawesome icons kit -->
+    <script src="https://kit.fontawesome.com/7f35089b8a.js" crossorigin="anonymous"></script>
+    <!-- Include Google Fonts link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+    <!-- Start header section -->
+    <header>
+        <!-- Start container -->
+        <div class="container">
+            <!-- Start logo section -->
+            <div class="logo-wrapper">
+                <a href="#" class="logo">My Logo</a>
```

Comment:
- [ ] Kindly personalize this, either by putting your own preferred logo or using your first name as appropriate. This will create professionalism. 🙂